src/webextension/list/components/item-list.css

```diff
@@ -19,3 +19,25 @@
   border-bottom: 1px solid #d7d7db;
   background-color: #ffffff;
 }
+
+.item:hover {
+  background-color: #ededf0;
+}
+
+.item:active {
+  background-color: #d7d7db;
+}
+
+.item[data-selected] {
+  background-color: #ebF5ff;
+  border-bottom-color: #99bff2;
+  box-shadow: 0 -1px 0 #99bff2;
+}
+
+.item:first-child[data-selected] {
+  border-top-color: #99bff2;
+}
+
+.item-list:focus > .item[data-selected] {
+  box-shadow: inset 0 1px 0 #0a84ff, inset 0 -1px 0 #0a84ff;
```

Comment:
I think this is actually going slightly against the spec. This gives an "inner" border on the top/bottom of the row, in addition to the regular border. The CSS that you have above for `.item[data-selected]` looks more-accurate (where you specify a `bottom-border-color` and a `box-shadow` to draw on top of the bottom border of the previous element).

I'm basing this on the spec [here](https://app.zeplin.io/project/59fb4868c5e7608593f76965/screen/5a85d57f54a506c53acf1d98), although it specifies a state the user can't (currently) get to. In the spec, the selected row and the focused row are different, but they must always be the same the way our code works right now. Still, it *looks* like the focus indicator is supposed to change the color of the borders between each row.

All that said, we *do* use an "inner" border to represent focus state in a lot of places in our UI (e.g. the buttons on the footer of the doorhanger), so doing it the way you have now would probably be consistent; we do usually use an inner border of 2px on *all* sides though.